src/main/java/racingcar/Cars.java

```diff
@@ -0,0 +1,23 @@
+package racingcar;
+
+import java.util.Collections;
+import java.util.List;
+
+public class Cars {
+    private List<Car> cars;
+
+    public Cars(final List<Car> cars) {
+        this.cars = Collections.unmodifiableList(cars);
+    }
+
+    public void move(RandomMove randomMove) {
+        cars.stream()
+                .filter(car -> randomMove.canMove())
+                .forEach(car -> car.move());
```

Comment:
method reference를 사용할 수 있겠네요!
```suggestion
                .forEach(Car::move);
```